homeassistant/components/refoss_rpc/config_flow.py

```diff
@@ -0,0 +1,310 @@
+"""Config flow for Refoss RPC integration."""
+
+from __future__ import annotations
+
+from collections.abc import Mapping
+from typing import Any, Final
+
+from aiorefoss.common import (
+    ConnectionOptions,
+    fmt_macaddress,
+    get_info,
+    get_info_auth,
+    mac_address_from_name,
+)
+from aiorefoss.exceptions import (
+    DeviceConnectionError,
+    InvalidAuthError,
+    MacAddressMismatchError,
+)
+from aiorefoss.rpc_device import RpcDevice
+import voluptuous as vol
+
+from homeassistant.config_entries import ConfigFlow, ConfigFlowResult
+from homeassistant.const import CONF_HOST, CONF_MAC, CONF_PASSWORD, CONF_USERNAME
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.aiohttp_client import async_get_clientsession
+from homeassistant.helpers.service_info.zeroconf import ZeroconfServiceInfo
+
+from .const import DOMAIN, LOGGER
+from .coordinator import async_reconnect_soon
+
+CONFIG_SCHEMA: Final = vol.Schema(
+    {
+        vol.Required(CONF_HOST): str,
+    }
+)
+
+
+INTERNAL_WIFI_AP_IP = "10.10.10.1"
+
+
+async def validate_input(
```

Comment:
```suggestion
async def async_validate_input(
```

Not really required, but this honours the naming convention